easybuild/easyblocks/s/scipion.py

```diff
@@ -0,0 +1,208 @@
+##
+# Copyright 2018 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing Scipion, implemented as an easyblock
+
+@author: Kenneth Hoste (Ghent University)
+@author: Ake Sandgren (HPC2N, Umea University)
+"""
+import os
+
+from distutils.version import LooseVersion
+from easybuild.framework.extensioneasyblock import ExtensionEasyBlock
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import copy, mkdir, symlink
+from easybuild.tools.modules import get_software_root, get_software_version
+from easybuild.tools.py2vs3 import configparser
+from easybuild.tools.run import run_cmd
+
+
+class EB_Scipion(ExtensionEasyBlock):
+    """Support for building/installing Scipion."""
+
+    def __init__(self, *args, **kwargs):
+        """Initialize Scipion-specific variables."""
+        super(EB_Scipion, self).__init__(*args, **kwargs)
+        # strip off 'scipion-*' part to avoid having everything in a subdirectory
+        self.unpack_options = '--strip-components=1'
+
+        # scipion.cfg file is in start_dir which isn't set until later
+        # so it gets initialized in configure_step instead.
+        self.cfgdir = ""
+        self.cfgfile = ""
```

Comment:
`self.cfgfile` is only used in `configure_step`, so no need for a class variable, just make it `cfgfile` in `configure_step`?

`self.cfgdir` is only used to define `self.cfgfile`, so no need for that at all, just use:

```python
cfgfile = os.path.join(self.builddir, 'config', 'scipion.conf')
```

in `configure_step`